tests/GatewayTest.php

```diff
@@ -88,6 +89,30 @@ function testCreatePurchaseOrder()
         $this->assertEquals($resp->body['request_payload'], $data);
     }
 
+    /**
+     * @group gateway
+     * request has authorization,
+     * is GET
+     * is to the correct url
+     */
+    function testGetPurchaseOrder()
+    {
+        $transport = new RequestToBodyMockTransport();
+        $gateway = new Gateway(self::HOST, array('transport' => $transport));
+        $gateway->setJwt($this->jwt);
+        $resp = $gateway->getPurchaseOrder();
+        $authorized_with_token = $resp->body['headers']['Authorization'];
```

Comment:
the convention I was using, I think based upon some PSR with php, or a common library used camelcase for local variables.

Reply:
Copied from prior test. Could certainly go through and do cleanup.